### DISCUSSION TITLE
Map rotate causes issues with zone cleaning

### DISCUSSION THREAD

**Discussion by geezer54** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#172] [Q&A] [answered]
First, I wanted to say how impressed I am with your new card.  Its quite a significant upgrade from your previous release!

When the map is rotated 90% via [Xiaomi-Cloud-Map-Extractor config](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#map-transformation-configuration), the selection box in zone cleaning does not work correctly. Mouse movement up causes selection box lower edge to move down and vice versa.  Also, the x,y coordinates appear to be exchanged.  FYI:  I'm using chrome browser on Windows 10.

https://user-images.githubusercontent.com/67194877/142241761-f3f7a701-3cb2-4473-9518-eeae0528dba3.mp4

**Comment by PiotrMachowski** [comment]
I have created an issue for this case: #173

**Comment by PiotrMachowski** [comment, marked as answer]
Problem should be solved in v2.0.3
